code/Controllers/ContentController.php

```diff
@@ -290,16 +290,6 @@ public function getMenu($level = 1)
         return new ArrayList($visible);
     }
 
-    /**
-     * @return ArrayList<SiteTree>
-     * @deprecated 5.4.0 Use getMenu() instead. You can continue to use $Menu in templates.
-     */
-    public function Menu($level)
-    {
-        Deprecation::noticeWithNoReplacment('5.4.0', 'Use getMenu() instead. You can continue to use $Menu in templates.');
-        return $this->getMenu($level);
-    }
```

Comment:
This isn't needed anymore. It existed because you couldn't pass args into getters from a template using `$Menu(2)` syntax.
I've resolved that - the args now get passed through to the getter.